services/src/main/scala/cromwell/services/metadata/impl/sns/AwsSnsMetadataServiceActor.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates.
```

Comment:
Not sure what the policy would be on re-asserting different copyright claims within this repo. Might need checking